repository.rosinstall

```diff
@@ -0,0 +1,4 @@
+- git:
+    local-name: sr_common
+    uri: https://{{github_login}}:{{github_password}}@github.com/shadow-robot/sr_common
```

Comment:
Please remove the login and password. They are not necessary for this public repository (you can check other examples for this).